openml-lightgbm/lightgbm-provider/src/test/java/com/feedzai/openml/provider/lightgbm/TestSchemas.java

```diff
@@ -202,4 +202,18 @@
             0,
             TestSchemas.NUMERICALS_SCHEMA_WITH_LABEL_AT_END.getPredictiveFields()
     );
+
+    /**
+     * Schema with two features with non-ASCII characters.
+     */
+    static final DatasetSchema SCHEMA_WITH_TWO_NON_ASCII_FEATURES = new DatasetSchema(
+            0,
+            ImmutableList.<FieldSchema>builder()
+                    .addAll(NUMERICALS_SCHEMA_WITH_TARGET_AT_START.getFieldSchemas())
+                    .add(new FieldSchema("cão", 5,
+                            new NumericValueSchema(false)))
+                    .add(new FieldSchema("test \uD801\uDC00", 6,
+                            new NumericValueSchema(false)))
```

Comment:
nitpick:
this could be in the same line

```
            ImmutableList.<FieldSchema>builder()
                    .addAll(NUMERICALS_SCHEMA_WITH_TARGET_AT_START.getFieldSchemas())
                    .add(new FieldSchema("cão", 5, new NumericValueSchema(false)))
                    .add(new FieldSchema("test \uD801\uDC00", 6, new NumericValueSchema(false)))
                    .build()
```